### PR TITLE
#162474707 implement all landing page functionality 

### PR DESCRIPTION
**Pull request by ucheg6**
#### What does this PR do?
implement all landing page functionality 
#### Description of Task to be completed?
- display author details where necessary in the landing page
- connect link to read single articles component
- connect link to creat article component
- create bookmark article saga and dispatch action to bookmark and unbookmark article

#### How should this be manually tested?
On command line ;
- git clone this repo
- npm install 
- Run `npm start:dev ` from the `haven-ah-frontend` directory

#### What are the relevant pivotal tracker stories?
[#162474707](https://www.pivotaltracker.com/story/show/162474707)

#### Screenshots (if appropriate)>
NONE

### CODE REVIEW

**Review by TheoOkafor — COMMENTED** [review]
This looks good @ucheg6

File: public/index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <meta http-equiv="X-UA-Compatible" content="ie=edge">
         <title>Authors Haven</title>
-        <base href="/"/>
+        <base href="/" />
```
Comment (seyi-adeleke) [inline public/index.html]: why this space?

File: public/styles/App.scss [hunk 2]
```diff
@@ -253,4 +253,5 @@ h1, h2, h3, h4, h5, h6 {
     background-image: url('../images/signup.png');
     background-size: cover;
     background-repeat: no-repeat;
-}
\ No newline at end of file
+}
+a { color: inherit; }
```
Comment (seyi-adeleke) [inline public/styles/App.scss:261]: Are you using this?

File: src/actions/bookmarkArticle.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  BOOKMARK_ARTICLE_REQUEST,
+  BOOKMARK_ARTICLE_SUCCESS,
+  BOOKMARK_ARTICLE_FAILURE,
+  UNBOOKMARK_ARTICLE_REQUEST,
+  UNBOOKMARK_ARTICLE_SUCCESS,
+  UNBOOKMARK_ARTICLE_FAILURE,
+} from '../actionTypes/bookmarkArticleActionTypes';
+
+export const bookmarkArticleAction = slug => ({
+  type: BOOKMARK_ARTICLE_REQUEST,
```
Comment (seyi-adeleke) [inline src/actions/bookmarkArticle.js:11]: I thought we agreed to have consistent actions? `type` and `payload `?

File: public/styles/App.scss (hunk 2 above)
Comment (ucheg6) [reply public/styles/App.scss:261]: Yeah I am

File: src/actions/bookmarkArticle.js (hunk 3 above)
Comment (ucheg6) [reply src/actions/bookmarkArticle.js:11]: Yeah. sorry about that. That has been implemented.

File: public/index.html (hunk 1 above)
Comment (ucheg6) [reply public/index.html]: I've removed it. Thanks

**Pull request merged**